pkg/specgen/winpath.go

```diff
@@ -0,0 +1,59 @@
+package specgen
+
+import (
+	"fmt"
+	"strings"
+	"unicode"
+
+	"github.com/pkg/errors"
+)
+
+func isHostWinPath(path string) bool {
```

Comment:
possible to get unittests on these functions to avoid simple regressions?

Reply:
@baude these are tested via specgenutil_test which I originally had to do for API access reasons. Although, I think those reasons may no longer apply. I can see if I can move them to specgen

Reply:
@baude ah yes, it creates an import cycle that was the other reason they had to be in the other package.